src/renderers/shared/reconciler/__tests__/ReactIdentity-test.js

```diff
@@ -107,12 +107,12 @@ describe('ReactIdentity', function() {
     var Wrapper = React.createClass({
 
       render: function() {
-        var span1 = <span ref="span1" key={key} />;
-        var span2 = <span ref="span2" />;
+        var s1 = <span ref="span1" key={key} />;
```

Comment:
`no-shadow`. On line 121 and 122:

```js
var span1 = instance.refs.span1;
var span2 = instance.refs.span2;
```

As a general rule, I prefer giving the shorter name to the more deeply-nested bit, so these instances got changed.